homepage/index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- adding google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Comfortaa&display=swap" rel="stylesheet">
+
+    <!-- adding google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
+
+
+
+    <!-- linking of css file -->
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Homepage</title>
+</head>
+
+<body>
+
+    <!-- section 1 nav bar -->
+    <header id="home">
+        <div class="packing">
+            <div class="logo">
+                <img src="/homepage/images/access_.png" alt="" width="100px" height="40px">
+            </div>
+            <nav class="navigation">
+                <ul>
+                    <li><a href="#AboutAccess">AboutAccess</a></li>
+                    <li><a href="#Courses">Courses</a></li>
+                    <li><a href="#i-campus"> <i>i</i>-campus</a></li>
+                    <li><a href="#Apply"><button>
+                                <p>Apply</p>
+                            </button></a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="heading">
+            <div class="icons">
+                <img src="/homepage/images/access_.png" alt="">
+                <!-- <p>access</p>
+                <span>_</span> -->
+            </div>
+            <div class="burger">
+                <div class="vector1"><img src="/homepage/images/Vector.png" alt=""></div>
+            </div>
+        </div>
+    </header>
+
+
+
+
+    <div class="banner-image">
+        <div class="choose"><span>Choose your Path Wisely</span>
+            <p>Make This Monsoon , A ladder to success. Weekend Courses on Web Development, Mobile App Development
+                and UX/UI
+                Design starting in June and July.</p>
+        </div>
+    </div>
+
+
+    <div class="why-access">
+        <div class="why-access-text">
+            <p>Why access?</p>
+        </div>
+        <div class="wrapper">
+            <div class="community">
+                <img src="/homepage/images/communitylearning.png" alt="">
+                <span>COMMUNITY LEARNING</span>
```

Comment:
Give Top and bottom margin for this heading - check design once

Reply:
done